bpf/process/bpf_fork.c

```diff
@@ -21,13 +21,23 @@ __attribute__((section("kprobe/wake_up_new_task"), used)) int
 BPF_KPROBE(event_wake_up_new_task, struct task_struct *task)
 {
 	struct execve_map_value *curr, *parent;
+	struct msg_clone_event msg;
+	u64 msg_size = sizeof(struct msg_clone_event);
 	u32 tgid = 0;
 
 	if (!task)
 		return 0;
 
 	tgid = BPF_CORE_READ(task, tgid);
 
+	/* Do not try to create any msg or calling execve_map_get
+	 * (that will add a new process in the execve_map) if we
+	 * cannot find it's parent in the execve_map.
+	 */
+	parent = __event_find_parent(task);
+	if (!parent)
```

Comment:
Would be nice to have a counter for this! I'm curious if  it will ever happen, when the second patch is also applied. 
That being said,  since we are not sending an event if we don't find the parent, there is no point in adding an entry to the map. So this patch makes sense to me as is.

Reply:
Yes, I have in mind to add a counter for that (and other cases in exec/exit events) to check if this happens in a follow-up PR.